data/sql/updates/pending_db_world/rev_1658828708862320500.sql

```diff
@@ -0,0 +1,3 @@
+--
+UPDATE `creature_template` set `npcflag` = 16|64 where `entry`= 16278;
```

Comment:
I'd prefer the use of 
```
`npcflag`=`npcflag`&~(repair flag)
```

Reply:
So it should be like 
``
UPDATE `creature_template` set `npcflag` = &~(80) where `entry`= 16278;
``
right? Could you tell me what '~' stands for, since I could not find it, thanks!

Reply:
No, it should be 
```sql
UPDATE `creature_template` set `npcflag` = `npc_flag`&~(4096) where `entry`= 16278;
```

Look at the link I left on my [comment](https://github.com/azerothcore/azerothcore-wotlk/pull/12544#issuecomment-1201696414) yesterday, it will clarify your questions.

Reply:
doing `field=field&~(flag)` basically does means field = field - flag the same way `field=field|(flag)` would be field = field + flag